src/components/application/Ezunpaywall.vue

```diff
@@ -1,32 +1,60 @@
 <template>
-  <v-container class="my-10">
+  <v-container >
     <v-row justify="center" align="center">
-      <v-col
-        cols="12"
-        md="4"
-      >
-        <v-img
-          style="max-width: 350px"
-          :src="require('../../../public/images/circle.png')"
-          class="ma-2 mx-auto"
-          alt="ezunpaywall-img"
-        />
+      <v-col cols="12" md="6" order="2" order-md="1">
+        <v-card>
+          <video
+            :src="require('../../../public/videos/ezunpaywall-demo.webm')"
+            type="video/webm"
+            class="ma-2 mx-auto"
+            style="max-width: 100%"
+            muted
+            loop
+            autoplay
+          />
+        </v-card>
       </v-col>
-      <v-col
-        cols="12"
-        md="8"
-      >
-        <p class="ma-2 text text-center text-justify">
-          {{ $t("ezunpaywall") }}
-        </p>
+      <v-col cols="12" md="6" order="1" order-md="2">
+        <v-row>
+          <v-col cols="12" :align="alignCentered">
```

Comment:
You could use [`align-*` class](https://v2.vuetifyjs.com/en/styles/flex/#flex-align) instead of a switch/case

```suggestion
          <v-col cols="12" class="align-center align-md-stretch">
```